### ISSUE TITLE
Migration from 2.0 to 2.1x documentation missing information

### ISSUE THREAD

**Issue by cutnicace** [containous/traefik#6378]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?
Wanted to migrate from traefik 2.0 to latest 2.1

### What did you expect to see?
Clear and detailed instruction what to edit for compatibility of existing traefik 2.0 configuration with traefik 2.1x.

### What did you see instead?
Found the migration documentation https://docs.traefik.io/migration/v2/ which states only two definitions need to be added (TraefikService and ClusterRole) **but it is not mentioned _where_ these definitons need to be made.**

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      2.0.5
Codename:     montdor
Go version:   go1.13.4
Built:        2019-11-14T18:10:56Z
OS/Arch:       #linux/arm
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

Do I even need to add these definitions if I only use a static traefic configuration with a file provider?

**Comment by ldez** [comment]
Hello,

The CRDs are for the Kubernetes provider: https://docs.traefik.io/v2.1/providers/kubernetes-crd/

You have to add it only if you are using our Kubernetes CRD provider.

**Comment by traefiker** [comment]
Closed by #6380.
